# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/l10n/L10nOverlays.cpp
# Commit: d88b294e0a5e
# Full Hash: d88b294e0a5e00a0ca739037ffa1452e39879382
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-03 22:01:19
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part2: Switch Localization class to use localization-ffi. r=emilio,nika
#   
#   Depends on D104788
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D104789
# ==============================================================================

diff -r e6bebff87544 -r d88b294e0a5e dom/l10n/L10nOverlays.cpp
--- a/dom/l10n/L10nOverlays.cpp	Tue Aug 03 05:52:00 2021 +0000
+++ b/dom/l10n/L10nOverlays.cpp	Tue Aug 03 05:52:00 2021 +0000
@@ -516,7 +516,7 @@
               DocumentFragment(aElement.OwnerDoc()->NodeInfoManager());
       nsContentUtils::ParseFragmentHTML(
           NS_ConvertUTF8toUTF16(aTranslation.mValue), fragment,
-          nsGkAtoms::_template, kNameSpaceID_XHTML, false, true);
+          nsGkAtoms::_template, kNameSpaceID_XHTML, false, true, 0);
       if (NS_WARN_IF(aRv.Failed())) {
         return;
       }